tests/composition/test_autodiffcomposition.py

```diff
@@ -466,7 +421,7 @@ class TestTrainingCorrectness:
         'eps, calls, opt, from_pnl_or_not, expected', [
             (100, 'single', 'adam', True, [[[0.09823965]], [[0.81092879]], [[0.78179557]], [[0.25593583]]]),
             (50, 'multiple', 'adam', True, [[[0.31200036]], [[0.59406178]], [[0.60417587]], [[0.52347365]]]),
-            (100, 'single', 'adam', False, [[[0.12697489]], [[0.74632817]], [[0.80712739]], [[0.28699516]]]),
+            (1, 'single', 'adam', True, [[[0.94790852]], [[0.95989353]], [[0.94048928]], [[0.95685814]]]),
```

Comment:
why do we need to test both `100-single-adam-True` and `1-single-adam-True`?
Why does this remove `single-adam-false`?
The `False` variant suffered more from lack of accuracy.

Reply:
Remove `param_init_pnl`, as it was deprecated. This should remove the issue with the False cases, as they were irrelevant anyways.

Reply:
so why does it keep `50-multiple-adam-False` ?

Reply:
whoops typo in previous comment -  meant to say "Remove**d**" instead of "Remove".
Changes are pushed - will squash fixups when everything is approved.